Projects/TEAM-2/Klopotenko/302/302/University.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace _302
+{
+    public class University
+    {
+        List<Student> student = new List<Student>();
+        public void AddStudent(string Name, int Course, int GradeMathematics, int GradePhysics, int GradePE)
+        {
+            bool isExist = true;
+            for (int i = 0; i < student.Count; i++)
+            {
+                if (Name == student[i].Name)
+                {
+                    Console.WriteLine($"Student {Name} is already exist!");
+                    isExist = false;
+                    break;
+                }
+            }
+            if (isExist)
+            {
+                student.Add(new Student(Name, Course, GradeMathematics, GradePhysics, GradePE));
+            }
+        }
+        public void ShowCourse(int Course)
+        {
+            int Students = 0;
+            for (int i = 0; i < student.Count; i++)
+            {
+                if (student[i].Course == Course)
+                {
+                    Console.WriteLine($"Name: {student[i].Name}, Course: {Course}, Math: {student[i].GradeMathematics}, Physics: {student[i].GradePhysics}, PE: {student[i].GradePE}");
+                    Students++;
+                }
+            }
+            if (Students == 0)
+            {
+                Console.WriteLine("No students on this course!");
+            }
+
+        }
+        public void ShowAverageSuccess(int Course, string Subject)
+        {
+            switch (Subject)
```

Comment:
Could you please try to calculate the average using not the switch\case but LINQ? It could be really great.

Reply:
I did it. But I don't know is it fully correct